.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -424,7 +424,8 @@ jobs:
 
       - name: set env variable(if x86_32 test needed)
         if: >
-          (matrix.test_option == '$DEFAULT_TEST_OPTIONS' || matrix.test_option == '$THREADS_TEST_OPTIONS')
+          (matrix.test_option == '$DEFAULT_TEST_OPTIONS' || matrix.test_option == '$THREADS_TEST_OPTIONS'
+           || matrix.test_option == '$WASI_TEST_OPTIONS')
```

Comment:
why are MULTI_MODULES_TEST_OPTIONS and SIMD_TEST_OPTIONS excluded here?

Reply:
SIMD isn't supported on x86-32 yet.

For multi module, maybe we can add it, one of the original considerations is that the total execution time of CIs is long, we didn't add all CI tests for x86-32.